dockerfiles/Dockerfile.arm32v7

```diff
@@ -13,21 +31,26 @@ RUN ./configure --system-curl
 RUN make
 RUN sudo make install
 
+# Set up build args
+ARG BUILDTYPE=MinSizeRel
```

Comment:
MinSizeRel is not supported. Don't try it.

Reply:
I was able to build and run with MinSizeRel, so it does work (even if not officially supported). What would be the recommended build flavor, keeping in mind this is meant to run on low-powered devices? Release?